package.json

```diff
@@ -29,10 +29,9 @@
   },
   "homepage": "https://github.com/OqixDevs/OqixTS#readme",
   "dependencies": {
-    "@discordjs/builders": "^0.15.0",
```

Comment:
No builder dependency is needed?

Reply:
Yes v14 has [builders included](https://discordjs.guide/additional-info/changes-in-v14.html#builders-are-now-included-in-v14)